modules/ibmz-configure-nbde-with-static-ip.adoc

```diff
@@ -0,0 +1,159 @@
+<3> Whether to enable or disable FIPS mode. By default, FIPS mode is not enabled. If FIPS mode is enabled, the {op-system-first} machines that {product-title} runs on bypass the default Kubernetes cryptography suite and use the cryptography modules that are provided with {op-system} instead.
+endif::ibm-z-kvm[]
+
+. Create a customized initramfs file to boot the machine, by running the following command:
++
+[source,terminal]
+----
+$ coreos-installer pxe customize \
+    /root/rhcos-bootfiles/rhcos-<release>-live-initramfs.s390x.img \
+    --dest-device /dev/sda --dest-karg-append \
+    ip=<ip-address>::<gateway-ip>:<subnet-mask>::<network-device>:none \
+    --dest-karg-append nameserver=<nameserver-ip> \
+    --dest-karg-append rd.neednet=1 -o \
+    /root/rhcos-bootfiles/<Node-name>-initramfs.s390x.img
+----
++
+[NOTE]
+====
+Before first boot, you must customize the initramfs for each node in the cluster and add PXE kernel parameters.
+====
+
+. Create a parameter file that includes `ignition.platform.id=metal` and `ignition.firstboot`.
++
+Example kernel parameter file for the control plane machine:
++
+ifndef::ibm-z-kvm[]
+[source,terminal]
+----
+rd.neednet=1 \
+console=ttysclp0 \
+coreos.inst.install_dev=/dev/dasda \ <1>
+ignition.firstboot ignition.platform.id=metal \
+coreos.live.rootfs_url=http://10.19.17.25/redhat/ocp/rhcos-413.86.202302201445-0/rhcos-413.86.202302201445-0-live-rootfs.s390x.img \
+coreos.inst.ignition_url=http://bastion.ocp-cluster1.example.com:8080/ignition/master.ign \
+ip=10.19.17.2::10.19.17.1:255.255.255.0::enbdd0:none nameserver=10.19.17.1 \
+zfcp.allow_lun_scan=0 \ <2>
+rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 \
+rd.zfcp=0.0.5677,0x600606680g7f0056,0x034F000000000000 \ <3>
+zfcp.allow_lun_scan=0 \
+rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 \
+rd.zfcp=0.0.5677,0x600606680g7f0056,0x034F000000000000
+----
+<1> For installations on DASD-type disks, add `coreos.inst.install_dev=/dev/dasda`. Omit this value for FCP-type disks.
+<2> For installations on FCP-type disks, add `zfcp.allow_lun_scan=0`. Omit this value for DASD-type disks.
+<3> For installations on DASD-type disks, replace with `rd.dasd=0.0.3490` to specify the DASD device.
+endif::ibm-z-kvm[]
+ifdef::ibm-z-kvm[]
+[source,terminal]
+----
+rd.neednet=1 \
+console=ttysclp0 \
+ignition.firstboot ignition.platform.id=metal \
+coreos.live.rootfs_url=http://10.19.17.25/redhat/ocp/rhcos-413.86.202302201445-0/rhcos-413.86.202302201445-0-live-rootfs.s390x.img \
+coreos.inst.ignition_url=http://bastion.ocp-cluster1.example.com:8080/ignition/master.ign \
+ip=10.19.17.2::10.19.17.1:255.255.255.0::enbdd0:none nameserver=10.19.17.1 \
+zfcp.allow_lun_scan=0 \
+rd.znet=qeth,0.0.bdd0,0.0.bdd1,0.0.bdd2,layer2=1 \
+rd.zfcp=0.0.5677,0x600606680g7f0056,0x034F000000000000
+----
+endif::ibm-z-kvm[]
++
+Write all options in the parameter file as a single line and make sure you have no newline characters.
```

Comment:
This can be a note or added to the step above. I would prefer a note consideirng the structure and reowrding it to be something like:
`Ensure that all options in the parameter file are in a single line, and have no newline characters.`

Reply:
I would like to keep this active voice. But having it as a note is a good suggestion.